blog/arquillian-cube-1.0.0.textile

```diff
@@ -0,0 +1,19 @@
+---
+date: 2017-03-24
+author: lordofthejars
+---
+
+p. New version of Arquillian Cube 1.0.0 has been released with one new feature and few bug fixes.
```

Comment:
It was my intention to do it globally, but we started rolling it per module basis, so maybe it's worth to mention?